xhr/client/src/main/scala/endpoints4s/xhr/EndpointsSettings.scala

```diff
@@ -0,0 +1,37 @@
+package endpoints4s.xhr
+
+import endpoints4s.Hashing
+
+import scala.annotation.nowarn
+
+final class EndpointsSettings(val host: Option[String]) extends Serializable {
```

Comment:
Some scaladoc would be great 😃

Reply:
By the way, I see in the tests that this `host` field is not really just a host. It also contains the port number, and even the protocol scheme (e.g., “http://localhost:8080” as opposed to just “localhost”). I wonder if there would be a better approach here. I like what we have in the http4s interpreter, although it requires maybe too much boilerplate?

Reply:
Looking at Akka HTTP client as an example, this could be named  `baseUrl` and be a native class, for example https://developer.mozilla.org/en-US/docs/Web/API/URL. How does that sound?

Reply:
Yes, sounds good! 👍

Reply:
Looking at the constructor of URL - https://developer.mozilla.org/en-US/docs/Web/API/URL/URL - maybe that is not such a good idea because relative URLs must have a base, where XHR does not require it. Seems that we can only build our own abstraction out of native classes for this, but should we do it?

Reply:
I think it is fine to use `String` here. I just wanted to raise the point that “host” might not be the best name, and that we should clarify in the docs what the users can put in this field.